src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -152,17 +162,19 @@ public Router vertxRouter(Vertx vertx,
 
         // add custom routers
         final String componentRoute = "/keyspace/:keyspace/table/:table/snapshots/:snapshot/component/:component";
-        final String defaultStreamRoute = V1_API_VERSION + componentRoute;
-        final String instanceSpecificStreamRoute = V1_API_VERSION + "/instance/:instanceId" + componentRoute;
-        router.route().method(HttpMethod.GET)
-              .path(defaultStreamRoute)
-              .handler(streamSSTableComponentHandler::handleAllRequests)
+        final String defaultStreamRoute = API_V1_VERSION + componentRoute;
+        router.get(defaultStreamRoute)
+              .handler(streamSSTableComponentHandler)
               .handler(fileStreamHandler);
 
-        router.route().method(HttpMethod.GET)
-              .path(instanceSpecificStreamRoute)
-              .handler(streamSSTableComponentHandler::handlePerInstanceRequests)
-              .handler(fileStreamHandler);
+        final String listSnapshotFilesRoute1 = "/keyspace/:keyspace/table/:table/snapshots/:snapshot";
+        final String listSnapshotFilesRoute2 = "/snapshots/:snapshot";
+
```

Comment:
1 and 2... Can we pick a more descriptive name? How about `listSnapshotFilesWithQualifiedTableNameRoute` and `listSnapshotFilesRoute`